tests/python_tests/test_vlm_pipeline.py

```diff
@@ -67,19 +67,26 @@ def streamer(word: str) -> bool:
         images = []
         for link in links:
             images.append(get_image_by_link(link))
-
-        ov_pipe.start_chat()
-
+        
+        # Test non chat mode
```

Comment:
FYI, there's a bug in phi3_v so it fails if there's no chat template and an image is preset. I'm working on it

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1670 fixes `FAILED tests/python_tests/test_vlm_pipeline.py::test_vlm_pipeline[model_id=katuni4ka/tiny-random-phi3-vision] - RuntimeError: Check 'images.empty() || m_history.empty()' failed at /runner/_work/openvino.genai/openvino.genai/src/src/cpp/src/visual_language/inputs_embedder.cpp:1485:`